firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -342,6 +342,10 @@ class BrowserViewController: UIViewController,
         return false
     }
 
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return currentTheme().type == .light ? .darkContent : .lightContent
```

Comment:
If we end up with this solution, I think we should do something similar to `getBarStyle()` on `ThemeType` instead of comparing to `.light` directly